includes/Modules/Idea_Hub.php

```diff
@@ -741,6 +758,40 @@ private function is_idea_post( $post_id ) {
 		return is_array( $this->get_post_idea( $post_id ) );
 	}
 
+	/**
+	 * Adds .googlesitekit-idea-hub__draft class to idea posts on the posts page.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $classes An array of post class names.
+	 * @param array $class An array of additional class names added to the post.
+	 * @param int   $post_id The post ID.
+	 * @return array An array of post class names.
+	 */
+	private function update_post_classes( $classes, $class, $post_id ) {
+		global $post_type;
+
```

Comment:
Let's use the screen object for this instead.
```suggestion
```

Reply:
Updated.